### DISCUSSION TITLE
Menu labels contain "ftom" not "from"

### DISCUSSION THREAD

**Discussion by timtjtim** [ololx/quick-symlink#22] [Announcements]
![Screenshot 2024-06-17 at 22 46 25](https://github.com/ololx/quick-symlink/assets/6292691/35fcee2a-4e01-4a6d-bfe3-5f164a14fb17)
![Screenshot 2024-06-17 at 22 46 23](https://github.com/ololx/quick-symlink/assets/6292691/d8bfb951-2c46-4f27-ac05-f8fe2b8ba539)

**Comment by ololx** [comment]
@timtjtim thank you for this issue and fix
